rdmo/projects/viewsets.py

```diff
@@ -737,6 +737,9 @@ def retrieve(self, request, *args, **kwargs):
 
     @action(detail=False, url_path='continue', permission_classes=(HasModelPermission | HasProjectPagePermission, ))
     def get_continue(self, request, pk=None, parent_lookup_project=None):
+        if not self.project.catalog.pages:
+            return Response(status=204)
```

Comment:
We might change that to 404 (I know that I used it in line 735 myself). My gut feeling says that 204 should be only used to indicate that the body of the request is empty (after `DELETE`). Displaying `FETCH_PAGE_ERROR: Not Found(404)` in the UI would also not be wring in this case. Lets talk later in person.